lib/openstack_query/runners/query_runner.py

```diff
@@ -155,7 +154,7 @@ def _run_paginated_query(
                 if i == self._LIMIT_FOR_PAGINATION - 1:
                     # restart the for loop with marker set
                     paginated_filters.update(
-                        {"marker": self._page_marker_prop_func(resource)}
+                        {"marker": self._prop_enum.get_prop_mapping(resource)}
```

Comment:
Plesae change the tests accordingly

Reply:
Attempting to do this change results in the following:

```
                if i == self._LIMIT_FOR_PAGINATION - 1:
                    # restart the for loop with marker set
                    paginated_filters.update(
>                       {"marker": self._prop_enum.get_prop_mapping()(resource)}
                    )
E                   TypeError: QueryRunnerTests.setUp.<locals>.<lambda>() missing 1 required positional argument: 'mock_obj'

lib/openstack_query/runners/query_runner.py:157: TypeError
```

Since it looks like we need an object here, as we're now calling the lambda instead simply returning it?